app.js

```diff
@@ -243,25 +243,15 @@ const userProfile = () => {
             if(href.includes(specialParameter)) href = href.substr(href.indexOf(specialParameter) + specialParameter.length, href.length);
             const parameters = getParameters(href);
             showAnimation();
+            
             if(parameters && parameters.token){
-                const response = await validateToken(parameters.token);
-                if(response.code === 200) {
-                    let obj = {
-                        335767902: (new Date(parseInt(user.metadata.a))).toISOString()
-                    }
-
-                    await storeResponse(obj);
-                }
+                await validateToken(parameters.token);
```

Comment:
what if the token can't be validated?

Reply:
if we aren't reacting to the result of validateToken() at all, it seems pointless to call it in the first place

Reply:
The `validateToken` function does more than validating the token. It also links uid to participant document. And current flow allows participant proceed, disregarding token valid or not. The only exception is 335767902, and this is why it is missing in some participant data.

Reply:
makes sense, thanks for clarifying!